doc-content/enterprise-only/openshift/configmaps-con.adoc

```diff
@@ -22,58 +22,58 @@ data:
 
 The operator uses ConfigMaps to configure and deploy the components. This includes all supported {PRODUCT} components such as {KIE_SERVER}, Smart Router, {CENTRAL} and service-related configuration such as for persistent volumes, build configuration, and routes. When ConfigMaps are edited manually, the operator uses the new values to create the deployments when the environment is reconciled.
 
-The {PRODUCT} operator can use the current version and the previous version of {PRODUCT} components concurrently, with ConfigMaps for each version, for example, 7.13.0 and 7.12.1.
+The {PRODUCT} operator can use the current version and the previous version of {PRODUCT} components concurrently, with ConfigMaps for each version, for example, 8.0.0 and 7.12.1.
 
-kieconfigs-7.12.1::
-This contains the `common.yaml` configuration file. For more information, see  https://github.com/kiegroup/kie-cloud-operator/blob/7.12.1-1/rhpam-config/7.12.1/common.yaml[`common.yaml`] on GitHub. You can configure the following components using this configuration file:
+kieconfigs-8.0.0::
+This contains the `common.yaml` configuration file. For more information, see  https://github.com/kiegroup/kie-cloud-operator/blob/8.0.0-1/rhpam-config/8.0.0/common.yaml[`common.yaml`] on GitHub. You can configure the following components using this configuration file:
```

Comment:
[GitHub Links] - Change URLs from /blob/ to /tree/, and versions as below:

https://github.com/kiegroup/kie-cloud-operator/blob/8.0.0-1/rhpam-config/8.0.0/
to:
https://github.com/kiegroup/kie-cloud-operator/tree/release-v7.13.x-blue/rhpam-config/8.0.0/
alternatively:
https://github.com/kiegroup/kie-cloud-operator/tree/release-v7.13.x-blue/rhpam-config/{ENTERPRISE_VERSION_LONG}/

[Question] - {PRODUCT_VERSION} and {PRODUCT_VERSION_LONG} point to {ENTERPRISE_VERSION} and {ENTERPRISE_VERSION_LONG} respectively. Which one should we use continuing forward?